app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryEffectHandler.kt

```diff
@@ -1,7 +1,7 @@
 package org.simple.clinic.bp.entry
```

Comment:
This commit is hard to read because mixes many things together:

- Moving the dependencies to the constructor
- Extracting methods
- Renaming variables

I think it would have been much easier to understand if each of these had been three separate commits.

Reply:
Noted, will be more conscious about these in the next PR.